tests/acceptance/cli/test_worklock_cli.py

```diff
@@ -230,7 +231,8 @@ def test_refund(click_runner, testerchain, agency_local_registry, token_economic
                     checksum_address=bidder,
                     worker_address=worker_address,
                     rest_host=MOCK_IP_ADDRESS,
-                    rest_port=select_test_port())
+                    rest_port=select_test_port(),
+                    db_filepath=tempfile.mkdtemp())
```

Comment:
Should this be an temp file, rather than a temp directory? It doesn't seem that we use `subdir=True` (https://lmdb.readthedocs.io/en/release/#lmdb.lmdb.open) when using `lmdb.open`.

Reply:
The `subdir` param is `True` by default. It could be turned off, but it would require that we create the directories first.